.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,71 @@
+---
+name: 🐛 Bug report
+description: Create a report to help us improve
+labels: bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        Before you continue filling out this report, please take a moment to check that your bug has not been [already reported on GitHub][issue search] 🙌
+
+        Remember to redact any sensitive information such as authentication credentials and/or license keys!
+
+        **Note:** If you are seeking community support or have a question, please consider starting a new thread via [GitHub discussions][discussions] or the [NGINX Community forum][forum].
+
+        [issue search]: ../search?q=is%3Aissue&type=issues
+        [discussions]: ../discussions
```

Comment:
Are these relative paths valid? A quick test of this path through the issue form on the `nginx/template-repo` yielded a 404.

![CleanShot 2025-03-28 at 11 32 19](https://github.com/user-attachments/assets/7891bd2c-71fe-46c8-8378-cc074ca5160d)

Reply:
Yup, they should be. If you select either of the options via https://github.com/nginx/template-repository/issues/new?template=bug_report.yml you should get linked to the right endpoint.